taskcluster/tc-build-utils.sh

```diff
@@ -247,7 +247,7 @@ do_deepspeech_netframework_build()
   MSBUILD="$(cygpath 'C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\MSBuild\Current\Bin\MSBuild.exe')"
 
   # We need MSYS2_ARG_CONV_EXCL='/' otherwise the '/' of CLI parameters gets mangled and disappears
-  # We build the .NET Client for .NET Framework v4.5,v4.6,v4.7
+  # We build the .NET Client for .NET Framework v4.5,v4.6,v4.7, netstandard2.1, netcoreapp3.1
```

Comment:
not here, please add `do_deepspeech_netstandard_build()` and `do_deepspeech_netcore_build()`

Reply:
done